js/skin.js

```diff
@@ -73,17 +73,40 @@ var Skin = React.createClass({
     });
   },
 
+  /**
+   * @public
+   * @description - returns the correct coordinates of events depending on the platform
+   * @param e - event
+   * @returns {object} - coordinates x, y
+   */
+  getCoords: function(e) {
```

Comment:
I think this should be moved to `Utils.js`. It's not directly related to `skin.js` logic

Reply:
fix